src/os/windows/win32.cpp

```diff
@@ -259,7 +259,7 @@ void CreateConsole()
 		return;
 	}
 
-#if defined(_MSC_VER) && _MSC_VER >= 1900
+#if defined(_MSC_VER)
```

Comment:
I think this `#if`, and whole `#else ... #endif`  section, could be removed too? This is Windows specific code with a CYGWIN guard.

Reply:
A `!`CYGWIN guard

Reply:
IIRC these `freopen` are still needed.

Reply:
Ah, MinGW strikes, of course.